src/index.js

```diff
@@ -315,4 +319,12 @@ Loadable.preloadReady = () => {
   });
 };
 
+Loadable.waitForLoad = () => {
```

Comment:
Can you add comments for these methods to explain usage? I don't know that it will be clear to folks otherwise. If I understand correctly, this differs from `preloadReady` and `preloadAll` in that it doesn't care what's ready.

It cares what has actually been used to render and only initializes those?

Reply:
Done!